test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -69,5 +64,21 @@ describe('distribute with fractions', function () {
     ],
   ];
 
-  tests.forEach(t => testDistributeResult(t[0], t[1]));
+  const multiStepTests = [
+
+    ['(2 / x +  3x^2) * (x^3 + 1)',
+      ['(2 * (x^3 + 1) / x + 3x^2 * (x^3 + 1))',
+        '(2 * (x^3 + 1) / x + (3x^5 + 3x^2))']
+    ],
+
+    ['(2x + x^2) * (1 / (x^2 -4) + 4x^2)',
```

Comment:
could you make one of them have a fancy numerator that need to get multiplied?

e.g. `(2x + x^2) * (3x^2 / (x^2 -4) + 4x^2)`

I'm not sure if this would end up getting multiplied actually... you can see how `(1 * (2x + x^2)` isn't resolved. Do you think it would be pedagogically useful to multiply it out as a substep? (vs how much extra code would it be :p )